app/views/questionnaire/components/universityDegreeCategory.scala.html

```diff
@@ -0,0 +1,19 @@
+@(questionnaireForm: Form[_root_.forms.EducationQuestionnaireForm.Data])(implicit lang: play.api.i18n.Lang)
+
+@import views.html.questionnaire.components.universityDegreeCategoryDetails
+@import views.html.widgets.{ checkbox, selectValueDifferentFromDisplayedText }
+
+<section>
+    @selectValueDifferentFromDisplayedText(questionnaireForm("universityDegreeCategory"),
+        models.view.questionnaire.UniversityDegreeCategory.seq.toList)(
+        "-- Select one --") {
+        @Messages("universityDegreeCategory.question")
+    }
+    <div class="form-group">
+    @checkbox(questionnaireForm("preferNotSay_universityDegreeCategory"), Some("universityDegreeCategory")) {
```

Comment:
This is the implementation of the checkbox:

@(field : play.api.data.Field, data: Option[String] = None)(labelContent: Html)

@import helpers.CSRSkinnyFieldConstructor._

@createId(id:String)=@{id.replace(" ", "_").replace("/", "_")}

@helper.input(field) { (id, name, value, args) =>
    <label for="@createId(id)" class="block-label" data-distarget='@data.getOrElse("")'>
        <input type="checkbox" id="@createId(id)" name="@name" value="true" @if(value.contains("true")){ checked } >
        @labelContent
    </label>
}

So, it is used,but not sure how the data-* tag works in this context.